sunpy/database/tests/test_database.py

```diff
@@ -440,6 +440,21 @@ def test_add_entry_from_hek_qr(database):
     # *something*
     assert len(database) > 1
 
+@pytest.mark.remote_data
+def test_download_from_hek_query_result(database, hek_qr, tmpdir):
+    assert len(database) == 0
+    database.download_from_hek_query_result(
+        hek_qr, path=str(tmpdir.join('{file}.fits')))
+    fits_pattern = str(tmpdir.join('*.fits'))
+    num_of_fits_headers = sum(
+        len(fits.get_header(file)) for file in glob.glob(fits_pattern))
+    assert len(database) == num_of_fits_headers > 0
+    for entry in database:
+        assert os.path.dirname(entry.path) == str(tmpdir)
+    database.undo()
```

Comment:
Was there a reason to run this command and check that it worked?

Reply:
No, I didn't need to run this to check if this worked. However I wanted to know whether any dependencies are handled before runtime.

Reply:
Sorry, I don't quite follow. What dependencies? Before the test is ran?

Reply:
I wanted to run test_database.py even though it doesn't print anything on the console, just to make sure my computer has the dependencies and python packages installed. 
I just wanted to run it without any errors and hence I was trying to install the packages in my venv.
Hope I answered your question. :)

Reply:
Sure. If we don't need those database undo/redo I think we should remove it.